schemas/external/repo/directory-page.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "http://ns.adobe.com/adobecloud/core/1.0/directory-page",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Directory Page Object",
+  "type": "object",
+  "description": "Directory page object contains paging information for related directory when directory result is paginated.",
+  "definitions": {
+    "page": {
+      "properties": {
+        "orderBy": {
+          "title": "Order by property",
+          "type": "string",
+          "description": "The property by which this page is ordered.\n"
+        },
+        "order": {
+          "title": "Sort order",
+          "type": "string",
+          "pattern": "^(asc|desc)$",
+          "description": "Sort order, either \"asc\" or \"desc\".\n"
+        },
+        "start": {
+          "title": "First value",
+          "type": "string",
+          "description": "The first value, in sorted order, of the orderby property on this page.\n"
+        },
+        "next": {
+          "title": "Next page start value",
+          "type": "string",
+          "description": "The start value for the next page.\n"
+        },
+        "property": {
+          "title": "Filter properties",
+          "type": "string",
+          "description": "The list of properties by which the result is filtered, if any.\n"
+        },
+        "type": {
+          "title": "Filter type value",
+          "type": "string",
+          "description": "The applied type filtering value, if any.\n"
+        },
+        "count": {
+          "title": "Number of items",
+          "type": "integer",
+          "description": "The number of items on this page.\n"
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/page"
+    }
+  ],
+  "required": [
+    "orderBy",
+    "order",
```

Comment:
The order property is optional and actually is not officially listed in the [Sorting, Paging, and Filtering](https://git.corp.adobe.com/pages/AdobeCloudPlatform/api-spec/#sorting-paging-and-filtering) Spec any more. Rather its functionality has been replaced by the `orderby`  property supporting multiple values with optional `+/-` to indicate sort order.

Not sure, maybe we should remove this property alltogether.

Reply:
@fmeschbe I do see 'order' listed in current spec, and I assumed that all the properties that do not have 'if any' are mandatory. I can removed or make this optional, and matching changes should be applied to spec as well.

Here's copy of current spec and page-object definition;
![image](https://user-images.githubusercontent.com/35572085/35286461-ee114d6e-0014-11e8-949f-9e0685eb6db3.png)

Reply:
Thanks. I suggest to make at least optional for now.